recipe-server/normandy/recipes/management/commands/update_actions.py

```diff
@@ -23,7 +23,8 @@ def add_arguments(self, parser):
 
     @transaction.atomic
     def handle(self, *args, **options):
-        action_names = settings.ACTIONS.keys()
+        action_names = list(settings.ACTIONS.keys())
+        action_names += list(settings.REMOTE_ACTIONS.keys())
         if options['action_name']:
             action_names = [name for name in action_names if name in options['action_name']]
```

Comment:
There is [a test that says it's OK to not match any action](https://github.com/peterbe/normandy/blob/7a289c7103b4b8df694a2032e42626f8fe751d9d/recipe-server/normandy/recipes/tests/test_commands.py#L140). I don't like this but didn't want to kick up a fuzz right now. 
If someone types `./manage.py update_actions preff-rolloutt` by accident it doesn't even complain. I think it should. Perhaps not very important.

Reply:
It felt easy to do since I was already in there messing around. Now you get:
```
▶ ./manage.py update_actions neverheardof
Raven is not configured (logging is disabled). Please see the documentation for more information.
INFO 2018-04-17 15:14:10,767 raven.contrib.django.client.DjangoClient Raven is not configured (logging is disabled). Please see the documentation for more information.
CommandError: No actions matching ['neverheardof']
```